# Array variable message of HumansFeat
Header header
HumansFeat[]  HumansDetected
